lodash/fp

**Describe the bug**
When installed globally, it does not add lodash and it is not possible to use it.
If you install it from the repository with `npm install -g` this does not happen.

**To Reproduce**

1. remove svgpack `npm uninstall @marsbased/svgpack -g`
2. Remove global lodash `npm uninstall lodash -g`
3. List global npm `npm ls -g `
4. New instalation with npm outside the repository `npm install @marsbased/svgpack -g`

**ERROR**
```
➜  svgpack git:(bug/lodash) svgpack test/assets
module.js:550
    throw err;
    ^

Error: Cannot find module 'lodash/fp'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/javierartero/.nvm/versions/node/v8.12.0/lib/node_modules/@marsbased/svgpack/src/pack.js:1:74)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
```

--- a/package.json
+++ b/package.json
@@ -19,6 +19,7 @@
   },
   "dependencies": {
     "globby": "^8.0.1",
+    "lodash": "^4.17.11",
     "minimist": "^1.2.0",
     "ora": "^3.0.0",
     "svgo": "^1.1.1",

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@marsbased/svgpack",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "repository": "git@github.com:MarsBased/svgpack.git",
   "author": "marsbased.com",
   "license": "MIT",

